android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/ScopedErrorRecoveryModule.kt

```diff
@@ -0,0 +1,28 @@
+package versioned.host.exp.exponent.modules.universal
+
+import android.content.Context
+import expo.modules.errorrecovery.ErrorRecoveryModule
+import expo.modules.errorrecovery.RECOVERY_STORE
+import host.exp.exponent.ExponentManifest
+import host.exp.exponent.kernel.ExperienceId
+import org.json.JSONObject
+import org.unimodules.core.ModuleRegistry
+
+class ScopedErrorRecoveryModule(context: Context, manifest: JSONObject, val experienceId: ExperienceId) : ErrorRecoveryModule(context) {
+  private val mCurrentVersion = manifest.getString(ExponentManifest.MANIFEST_SDK_VERSION_KEY) ?: ""
+
+  override fun onCreate(moduleRegistry: ModuleRegistry) {
+    mSharedPreferences = context.applicationContext.getSharedPreferences("$RECOVERY_STORE.$mCurrentVersion", Context.MODE_PRIVATE)
```

Comment:
Shouldn't we call `super.onCreate` in case the underlying class does something more than just set `mSharedPreferences` value?